mule-gateway/modules/ROOT/pages/mule-gateway-capabilities-mule4.adoc

```diff
@@ -122,9 +122,9 @@ This defines a resource-level pointcut for your API. In this example, the policy
 
 == Enabling Mule Gateway capabilities
 
-To take advantage of Mule Gateway capabilities, you need to establish a communication channel between the Mule runtime and an organization created in a particular plane (ie. US control plane, EU control plane) or Mule Private Cloud Edition.
+To take advantage of Mule Gateway capabilities, you need to establish a communication channel between Mule runtime and an organization created in a particular plane (ie. US control plane, EU control plane) or Mule Private Cloud Edition.
 
-To configure the Mule runtime in such a way, you need to specify your Anypoint organization credentials. +
+To configure Mule runtime in such a way, you need to specify your Anypoint organization credentials. +
```

Comment:
Those plusses at the end of a line of a paragraph (not in ordered or unordered list) drive me crazy bc the paragraph can look really weird and non-standard. 

word word word word. 
word word word word word word word word.

instead of

Reply:
word word word word. 

word word word word word word word word.

or 

word word word word. word word word word word word word word.